src/System.Runtime.Numerics/src/System.Runtime.Numerics.csproj

```diff
@@ -29,14 +29,19 @@
     <Compile Include="$(CommonPath)\System\Globalization\FormatProvider.Number.cs">
       <Link>System\Globalization\FormatProvider.Number.cs</Link>
```

Comment:
FormatProvider.Number.cs may benefit from ValueStringBuilder too. It is similar to `Number.Formatting.cs` in CoreCLR.

Reply:
Yup.  That's necessary in support of https://github.com/dotnet/corefx/issues/25336.